buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestTestPlugin.groovy

```diff
@@ -48,5 +49,12 @@ public class RestTestPlugin implements Plugin<Project> {
         integTest.group = JavaBasePlugin.VERIFICATION_GROUP
         integTest.mustRunAfter(project.precommit)
         project.check.dependsOn(integTest)
+
+        project.tasks.withType(JavaCompile) {
```

Comment:
I don't think this is actually needed, because StandaloneTestPlugin is required, which also has this. In fact, I think what is here now will cause -proc:none to be in the compile args twice?